backend/app/models/visualizations.py

```diff
@@ -0,0 +1,46 @@
+from datetime import datetime
+from typing import Optional, List
+
+from beanie import Document, View, PydanticObjectId
+from pydantic import BaseModel, Field
+
+from app.models.files import FileContentType
+from app.models.users import UserOut
+
+
+class VisualizationsBase(BaseModel):
```

Comment:
We use singular for other models. Also maybe we should call this `VisualizationConfig` to be more specific.